src/components/icon/Icon.vue

```diff
@@ -25,17 +34,9 @@
              * internal icons are always MDI.
              */
             newIcon() {
-                if (!this.both) {
-                    if (this.newPack === 'mdi') {
-                        return `${this.newPack}-${this.icon}`
-                    } else {
-                        return `fa-${this.icon}`
-                    }
-                }
-
                 return this.newPack === 'mdi'
                     ? `${this.newPack}-${this.icon}`
-                    : `fa-${this.getEquivalentIconOf(this.icon)}`
+                    : this.addFAPrefix(this.getEquivalentIconOf(this.icon))
```

Comment:
I think it isn't correct before when pack is FontAwesome you're adding always the equivalent icon.